.github/workflows/basic.yml

```diff
@@ -1,6 +1,11 @@
 # Based on https://github.com/actions-rs/example/blob/master/.github/workflows/quickstart.yml
 
-on: pull_request
+on: 
+  pull_request:
+  push:
+    branches:
+      - 'main'
+      - 'releases/**'
```

Comment:
The `s` here was confusing to me. I thought this was a mistake, because I thought standard was `release/**`, but I see on axelar-core we do use `releases/**`